db/migrate/20190403210056_add_location_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocationTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :locations
```

Comment:
I only added this for the sake of getting the mock to work. Future migration will add for real

Reply:
What will be the purpose of the table? Are we saving locations here (vs. metadata fields)?

Reply:
I think we probably should treat location separately, even though it breaks our old MetadataField mold. In the MetadataField approach we would have country/state/county/city/lat/lon as Metadatum on the sample, but then it'd be hard to query "what are the coordinates for this place", right? Maybe we can have hooks that update collection_location to be a display_name for a simple value, and then an associated record in Locations table for more info?

Reply:
Actually just deleted this. Don't need it "just for the stub"